nss-cohort-36/keahua-arboretum-net-surfers
# Jl first take

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-first-take`
4. Biome parent class created with interfaces for all biomes included.  individual biomes created and tested by importing into index and run from terminal
5.  No other functionality added to add or test for animals - that's next step
6.  each biome imported into __init__ file in environements

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
